scss/_vars.scss

```diff
@@ -1,6 +1,7 @@
-$cyan: #82ECEC;
-$pink: #EA668E;
-$lavender: #A88AA8;
-$light-lavender: #DED2E6;
-$light-grey: #E5E5E5;
-$grey: #C4C4C4;
+$cyan: #82ecec;
```

Comment:
these colors are slightly different from the home pg colors -- cyan & greys are fine but can u update the others to match the other pages:
pink: #f27aaa
lavender: #b5a1c4
light-lavender: #cca8c1
spons-blue: #6fcbdc